src/UI/CreditCardTab.java

```diff
@@ -328,7 +340,19 @@ private void initializeCreditCardModifyTab(JTabbedPane creditCardTabbedPane) {
 			public void actionPerformed(ActionEvent e) {
 				if(creditCardModifyCurNicknameInput.getText().isEmpty()) {
 					JOptionPane.showMessageDialog(frame, "Enter a credit card to modify");
-				}else {
+				} else if (!Validation.validateCardNumber(creditCardModifyNumberInput.getText())) {
+					
+					JOptionPane.showMessageDialog(frame, "Card number must be in the form ################ and be between 13-16 numbers long");
+					
+				} else if (!Validation.validateExpirationDate(creditCardModifyExpDateInput.getText())) {
+					
+					JOptionPane.showMessageDialog(frame, "Exp date must in one of the forms: M/YY, M/YYYY, MM/YY, MM/YYYY");
+					
+				} else if (!Validation.validateCvv(creditCardModifyCVVInput.getText())) {
+					
+					JOptionPane.showMessageDialog(frame, "Cvv must be in one of the forms: ### or ####");
```

Comment:
these validations are the same code with just creditCardAdd variables and creditCardModify variables. To have less code duplication what do you think about moving the validation code to a different method in credit card that takes in the credit card number input text, the expiration input text, and the cvv input text. That way both of these sections could just call the method with their respective input